CMakeLists.txt

```diff
@@ -77,10 +78,14 @@ if(UAGENT_P2P_PROFILE)
 endif()
 
 if(UAGENT_FAST_PROFILE)
-    set(_fastdds_version 2.0.1)
-    set(_fastdds_tag 2.0.x)
+    if(UAGENT_FAST_PROFILE AND NOT UAGENT_USE_SYSTEM_FASTDDS)
+        set(_fastdds_version 2.0.0)
+    else()
+        set(_fastdds_version 2.0.2)
+        set(_fastdds_tag v2.0.2)
+        set(_foonathan_memory_tag c619113) # This tag should be updated every time it gets updated in foonathan_memory_vendor eProsima's package
```

Comment:
This should be out of this `if` statement. And also check that the if statement in line 80 is duplicated.

Reply:
My fault.